tests/Unit/ProjectCodeTest.php

```diff
@@ -77,6 +77,8 @@ public function testProductionClassesHaveUnitTests()
                 Provider\ImageProcessorProvider::class,
                 Provider\ResetEmailerServiceProvider::class,
                 Provider\SentinelServiceProvider::class,
+                Provider\SwiftMailerServiceProvider::class,
+                Provider\TalkEmailerServiceProvider::class,
```

Comment:
`TalkEmailerServiceProvider` is gone, isn't it?

Reply:
Good catch, fixed